lib/beaker/host/mac/pkg.rb

```diff
@@ -43,7 +43,7 @@ def upgrade_package(name, _cmdline_args = '')
 
   # Examine the host system to determine the architecture
   # @return [Boolean] true if x86_64, false otherwise
-  def determine_if_x86_64
+  def determine_if_x86_64 # rubocop:disable Naming/VariableNumber
```

Comment:
https://docs.rubocop.org/rubocop/cops_naming.html#namingvariablenumber says `AllowedIdentifiers` already allows `x86_64` so I wonder why it's not working.

Reply:
I tested a bit around and the default isn't working, also `AllowedIdentifiers: ['x86_64']` isn't working. I will test a newer rubocop and then raise an upstream issue.

Reply:
`AllowedIdentifiers` is only for whole method names. It works when we set `AllowedPatterns`